### PR TITLE
Add Events Handler to the Component Class

### PR DESCRIPTION
**Pull request by Abdona**
Add event handler and bind it with the calculator buttons.

### CODE REVIEW

**Comment by DeliceLydia** [comment]
# Status: invalid

# Hi @Abdona,

I am sorry I can not continue to review your project because I am not able to run it
locally it seems like they some missing dependencies, this is the error am getting

error

Please fix it and submit another CRR

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,31 +1,50 @@
...
-        <button>6</button>
-        <button className="buttoperation">-</button>
-        <button>1</button>
-        <button>2</button>
-        <button>3</button>
-        <button className="buttoperation">+</button>
-        <button className="zeroitem">0</button>
-        <button>.</button>
-        <button className="buttoperation">=</button>
+        <input className="resultscreen" readOnly placeholder={this.state.total} />
```
Comment (MahmoudBakr23) [inline src/components/Calculator.js]: - [x] Kindly double-check on the rendered values to the screen of the calculator, it seems that the numbers aren't rendering to the screen while operating. Only the results are! Please let's work on that 🙏 😄

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Abdona 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct
- Descriptive PR headline, summary, and commit messages

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [x] Kindly update the README file to be more consistent with the project 😄 
You can refer to this great [template](https://github.com/microverseinc/readme-template) to do so!

- [x] Please fix the MIT broken link for a completely professional README file and repo 😄

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,31 +1,85 @@
...
-        <button>1</button>
-        <button>2</button>
-        <button>3</button>
-        <button className="buttoperation">+</button>
-        <button className="zeroitem">0</button>
-        <button>.</button>
-        <button className="buttoperation">=</button>
+        <input className="resultscreen" readOnly placeholder={this.state.screen} />
+        <button onClick={this.calchandler}>AC</button>
+        <button onClick={this.calchandler}>+/-</button>
```
Comment (MahmoudBakr23) [inline src/components/Calculator.js:51]: - [x] Please fix the issue with this button
_See the image down below._ 🙏 😄 
Screenshot

File: src/components/Calculator.js [hunk 3]
```diff
@@ -1,31 +1,85 @@
...
+    //   this.setState({
+    //     screen: 0, total, next, operation,
+    //   });
+    // } else {
+    //   // this.num += e.target.innerText;
+    //   const { total, next, operation } = calculate(this.state, e.target.innerText);
+    //   alert(total);
+    //   // this.value = total;
+    //   // alert(this.value);
+    //   this.setState({
```
Comment (MahmoudBakr23) [inline src/components/Calculator.js]: - [x] Please remove any commented-out code through your files for cleaner and more optimized code 😄 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @Abdona 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [x] I highly recommend attaching a screenshot of your app to the README file.

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (DeliceLydia) [reply src/components/Calculator.js:51]: Well addressed this!

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved🎉 🎉 

## Hi @Abdona, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional

- I was failing to run `build` because you did not add `npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x  @babel/preset-react@7.x @babel/preset-react@7.x` in your dependencies, please add them before merging



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**